README.md

```diff
@@ -33,16 +33,46 @@ fmt.Println(lang)
 lang := enry.GetLanguage("foo.cpp", "<cpp-code>")
 ```
 
-Developmemt
+Development
 -----------
-*enry* re-uses parts of original [linguist](https://github.com/github/linguist) especially data in `languages.yml` to generate internal data structures. In oreder to update to latest upstream run
+*enry* re-uses parts of original [linguist](https://github.com/github/linguist) especially data in `languages.yml` to generate internal data structures. In order to update to latest upstream run
 
     make clean code-generate
 
 To run the tests
 
     make test
 
+Benchmark
```

Comment:
Minor: `Benchmarks` or `Benchmarking` may be better, since it's not a single benchmark.